src/router/routes.js

```diff
@@ -44,6 +44,12 @@ export const defaultRoutes = {
       component: () => import(/* webpackChunkName: 'landing' */ 'pages/NewAsset.vue'),
       meta: {}
     },
+    {
+      name: 'recruiters',
+      path: 'r',
+      component: () => import(/* webpackChunkName: 'landing' */ 'pages/Recruiters.vue'),
+      meta: {}
+    },
```

Comment:
Here you can also map to newAsset, and detect `this.$route.name` in this page to know if it `newAsset` or `recruiters`, to reuse the same component and avoid duplicating code.

```
{
      name: 'recruiters',
      path: 'r',
      component: () => import(/* webpackChunkName: 'landing' */ 'pages/NewAsset.vue'),
      meta: {}
    },
```

Reply:
I did Thank you